pkg/controller/che/create.go

```diff
@@ -132,6 +133,29 @@ func (r *ReconcileChe) CreateNewRoleBinding(instance *orgv1.CheCluster, roleBind
 	return nil
 }
 
+// CreateNewJob deploys new instance of given job
+func (r *ReconcileChe) CreateNewJob(instance *orgv1.CheCluster, job *batchv1.Job) error {
```

Comment:
Create a `syncJobToCluster` function with the same logic as for pvc, ingress, route ...

Reply:
I think it is better to keep ability to create an object (in our case job) configuration and then separately flush it to cluster. Because we want to have universal methods, but we never know for sure which aspects of it we may want to change in the future. Changing the sync* method signature is not good as it will affect other consumers. Updating object in runtime is worse or even impossible.
I may move this method into `job.go` and rename it to match `sync*` pattern, but I don't want to loose the flexibility here.